src/App.js

```diff
@@ -1,10 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import style from './App.module.scss';
+import store from './store/index';
```

Comment:
why you import a store?

please read how to use redux store in components and dispatch actions!!!